core/controllers/topic_viewer.py

```diff
@@ -73,9 +73,15 @@ def get(self, topic_name):
             summary.to_human_readable_dict() for summary
             in additional_story_summaries]
 
+        skill_ids = topic.get_all_skill_ids()
```

Comment:
Here, no need to get all skill ids, just the uncategorized skill ids is fine,  since the other skill ids are in the various subtopics. 
In the frontend we would be surfacing only subtopics as different 'concepts' to the learners, so subtopics are fine as a source of skill ids in the frontend.

Reply:
Okay thanks @aks681 . :)